files/meethyde/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "cdn",
+  "repo": "meethyde/cdn",
+  "files": {
+    "include": ["meethyde.min.js", "meethyde.min.css", "meethyde.png"]
```

Comment:
This wont work. Since the folder changes for every version our bot cant take the files https://github.com/MeetHyde/cdn/tree/master/0.1

You need to have the same file structure and use Github to tag your versions